pom.xml

```diff
@@ -158,7 +158,7 @@
         <dependency>
             <groupId>uk.gov.ons.ctp.product</groupId>
             <artifactId>casesvc-api</artifactId>
-            <version>10.49.30</version>
+            <version>10.49.32</version>
```

Comment:
I'm probably missing context (or coffee) but I assume there's a reason it's skipped `.31`?

Reply:
`.31` probably contained stuff we ended up not needing. We're also not very good at keeping our API versions up to date.